pypeit/core/coadd.py

```diff
@@ -2156,6 +2159,10 @@ def combspec(waves, fluxes, ivars, gpms, sn_smooth_npix,
         Boolean mask for stacked
         spectrum on wave_stack wavelength grid. True=Good.
         shape=(ngrid,)
+    nocoadd_exp : `numpy.ndarray`_
```

Comment:
This should be changed to a list (since the inputs are list) of length next, which is True if the exposure was used and False if the exposure was not used. That is more aligned with the structure of this code.